### PR TITLE
chore(release): update RBAC to delivery bug fixes for RHDH 1.3.1

### PR DESCRIPTION
**Pull request by AndrienkoAleksandr**
### What does this pull request fixes:

Fixes https://issues.redhat.com/browse/RHIDP-4069

### CODE REVIEW

File: .changeset/fuzzy-dryers-lie.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+"@janus-idp/backstage-plugin-audit-log-node": major
+"@janus-idp/backstage-plugin-rbac-backend": major
+"@janus-idp/backstage-plugin-rbac-common": major
+"@janus-idp/backstage-plugin-rbac-node": major
+"@janus-idp/backstage-plugin-rbac": major
```
Comment (PatAKnight) [inline .changeset/fuzzy-dryers-lie.md:6]: These should be consider patch instead of major

File: plugins/rbac-backend/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@janus-idp/backstage-plugin-rbac-backend",
-  "version": "4.10.6",
+  "version": "4.12.2",
```
Comment (PatAKnight) [inline plugins/rbac-backend/package.json:3]: From my understanding, we don't need this change. Changesets should make this change for us after we merge this pr

**Review by PatAKnight — COMMENTED** [review]
I believe that there are a number of unnecessary changes in this PR. We just need to backport the [fix ownerRefs](https://github.com/janus-idp/backstage-plugins/pull/2228/files) and [allow to disable group inheritance](https://github.com/janus-idp/backstage-plugins/pull/2143/files) PRs. Is there any reason that we need to include the changes to audit log, rbac common, rbac node, and rbac plugins?

File: plugins/rbac-backend/package.json (hunk 2 above)
Comment (AndrienkoAleksandr) [reply plugins/rbac-backend/package.json]: Ok, I removed this change. So next version for RHDH 1.3.1 will be 4.10.7?

**Pull request merged**